src/setup.js

```diff
@@ -17,23 +17,23 @@ function generateConfig() {
   return Client.discover()
     .then(device => {
       const { ip } = device;
-      const appMap = {};
+      const inputs = [];
       return device
         .apps()
         .then(apps =>
```

Comment:
Do the apps come back in the same order every time?

Reply:
They seem to.